registry/configuring-registry-operator.adoc

```diff
@@ -0,0 +1,33 @@
+[id=configuring-registry-operator’]
+= Configuring the Image Registry Operator in {product-title}
+include::modules/common-attributes.adoc[]
+:context: configuring-registry-operator
+toc::[]
+
+The Image Registry Operator installs a single instance of the {product-title}
+registry. It manages all configuration of the registry including setting up
+registry storage.
+
+[NOTE]
+====
+Storage is only automatically configured when you install on AWS.
```

Comment:
I'd s/AWS/Amazon Web Services

Do you have an issue to add, or refer to, the image registry storage configuration modules when they exist?